src/Castle.Facilities.Logging/LoggingFacility.cs

```diff
@@ -74,6 +76,7 @@ public LoggingFacility()
 		///   Initializes a new instance of the <see cref="LoggingFacility" /> class.
 		/// </summary>
 		/// <param name="loggingApi"> The LoggerImplementation that should be used </param>
+		[Obsolete("A logger factory implementation type should be provided via LogUsing<T>(), this will be removed in the future.")]
```

Comment:
There are currently 131 Obsolete attributes in the code base. Can't we just delete this altogether? Assuming we kick the version up to v5.